ansible/roles/tensorrt/tasks/main.yaml

```diff
@@ -16,6 +16,8 @@
   ansible.builtin.apt:
     name:
       - libcudnn8-dev={{ cudnn_version }}
+      - libnvinfer-headers-dev={{ tensorrt_version }}
+      - libnvinfer-headers-plugin-dev={{ tensorrt_version }}
```

Comment:
These need to be updated in the role's readme as well.

Also how is this related to:
- https://github.com/autowarefoundation/autoware/pull/4576 ?